src/api/app/components/bs_request_activity_timeline_component.rb

```diff
@@ -4,9 +4,9 @@
 # It merges the BsRequestCommentComponent and the BsRequestHistoryElement to
 # provide a merged timeline of events.
 class BsRequestActivityTimelineComponent < ApplicationComponent
-  attr_reader :bs_request, :creator, :timeline
+  attr_reader :bs_request, :creator, :timeline, :request_reviews
 
-  def initialize(bs_request:)
+  def initialize(bs_request:, request_reviews: [])
```

Comment:
You already pass around the `BsRequest`, you also know the `Review`s associated to it...

Reply:
true, I will adapt it

Reply:
@hennevogel I now remember why I went this route, we fetch the reviews for non staging projects here in the controller https://github.com/openSUSE/open-build-service/blob/2e4f9e1673bd1200748cad0382a80e31dbc2c3ff/src/api/app/controllers/webui/request_controller.rb#L538 there we have to first get the target project and then going through some scope to actually get the reviews. I wanted to avoid doing all these queries again the the view component. I could refactor this a bit, or do the fetching twice. But if its okay for you I would try to make this nicer in a follow up

Reply:
I can at least make the naming a bit nicer for now, so its clear I hand over the reviews for non staging projects

Reply:
I'm fine with pushing this into a follow up card.

Reply:
okay, tracked now in this card https://trello.com/c/cdPzPW7B/137-refactor-how-we-receive-reviews-for-non-staging-project